prosody/rootfs/defaults/prosody.cfg.lua

```diff
@@ -143,9 +148,13 @@ authentication = "internal_plain"
 --  Logs info and higher to /var/log
 --  Logs errors to syslog also
 log = {
-	{ levels = {min = "info"}, to = "console"};
+	{ levels = {min = "{{ $LOG_LEVEL }}"}, to = "console"};
 }
 
+{{ if .Env.GLOBAL_CONFIG }}
```

Comment:
please document this and pass it in docker-compose

Reply:
done